swupd/create_manifests.go

```diff
@@ -205,7 +205,7 @@ func processBundles(ui UpdateInfo, c config, numWorkers int) ([]*Manifest, error
 		oldM.sortFilesName()
 		changedFiles, added, deleted := bundle.linkPeersAndChange(oldM, ui.minVersion)
 		// if nothing changed, skip
-		if changedFiles == 0 && added == 0 && deleted == 0 && !changedIncludes {
+		if changedFiles == 0 && added == 0 && deleted == 0 && !changedIncludes && bundle.Name != "full" {
```

Comment:
Do you know why mixer crashes without full?

Reply:
It crashes if there's no manifests in a list of manifests in create_manifests. But anyway, we need to create the full manifest for this release even if it hasn't changed, right?